File: 프로그래머스/lv0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    price : int
    answer = 0
    
    if 300000 > price >= 100000:
        answer = price * 0.95
    elif 500000 > price >= 300000:
        answer = price * 0.9
    elif price >= 500000:
        answer = price * 0.8
    else:
        answer = price
    
    return int(answer)

File: 프로그래머스/lv0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        if(price >= 100000 && price <300000){
            return (int)(price *(1 - 0.05F));
        } else if (price >= 300000 && price < 500000) {
            return (int)(price *(1 - 0.1F));
        } else if (price >= 500000 && price <= 1000000) {
            return (int)(price *(1 - 0.2F));
        }else{
            return price;
        }
    }
}